### DISCUSSION TITLE
Include field for basionyms to improve discoverability of type specimens

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#500] [Wish List]
Suggestion made by a member of the Intermountain Regional Herbarium Network:

As currently structured, Symbiota data entry screens only allow for a single scientific name. For type specimens, the original name (basionym) is often different from the currently accepted scientific name. As types are very important for taxonomic research, it is very important for researchers to be able to search on the original scientific name of the type specimen. It is equally important to have the current name available too. One solution would be to have a separate scientific name field just for basionyms of type specimens.  It would also be useful to have an accompanying text field in the "Latest identification" section for taxonomic comments. Such a field could be used to explain the current name of the type specimen, cite the original literature source of the type collection, and other relevant comments.  Such a field might also be used for other types of species, such as hybrids, to record the parental taxa.  Or it might be useful for taxonomic comments that are otherwise put in the Description or Notes field near the bottom of the data entry form.  Sometimes this information is incorrectly placed in the Identification Qualifier field

**Comment by ljwalker** [comment]
Possible solutions to this request could relate to https://github.com/BioKIC/symbiota-docs/issues/155.

**Reply by fbungartz** [reply]
Just adding a "field" for basionym doesn't solve anything. Designating names as basynyms requires a conceptual change how the taxonomic thesaurus actually works.

**Comment by fbungartz** [comment]
> Possible solutions to this request could relate to https://github.com/BioKIC/symbiota-docs/issues/155.

Link doesn't work.
